cvat-core/src/server-proxy.js

```diff
@@ -924,14 +924,24 @@
                 return createdTask[0];
             }
 
-            async function getJob(jobID) {
+            async function getJobs(jobID = null, filter = {}) {
```

Comment:
`getTasks` and `getProjects` accept only one argument. Maybe this function can be made the same way?

Reply:
Updated, thx